src/main/replay_main.ml

```diff
@@ -37,28 +37,35 @@ let replay_tlogs tlog_dir tlf_dir db_name end_i =
       | _ -> Lwt.return ()
   in
   let t () =
+    let open Tlog_map in
     Lwt.catch
       (fun () ->
-         S.make_store ~lcnum:1024 ~ncnum:512 db_name >>= fun store ->
-         let cio = S.consensus_i store in
-         let cios = Log_extra.option2s Sn.string_of cio in
-         console "store @ %s" cios >>= fun () ->
-         begin
-           match
-             end_i with
-             | None ->
-               begin
-                 Tlc2.get_last_tlog tlog_dir tlf_dir >>= fun (_new_c,fn) ->
-                 Tlc2._validate_one fn "" ~check_marker:false >>= fun (last, _index) ->
-                 let i =
-                   match last with
-                     | None -> Sn.start
-                     | Some e -> Sn.succ (Tlogcommon.Entry.i_of e)
-                 in
-                 Lwt.return i
-               end
-             | Some i -> Lwt.return i
-         end
+        TlogMap.make tlog_dir tlx_dir "node_name???" ~check_marker:false
+        >>= fun (tlog_map,_,_) ->
+        S.make_store ~lcnum:1024 ~ncnum:512 db_name >>= fun store ->
```

Comment:
This replay doesn't use a config. It feels a bit heavy to demand a full arakoon config just for this.